src/main/java/uk/gov/justice/digital/delius/data/api/UploadedDocumentCreateResponse.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class UploadedDocumentCreateResponse {
```

Comment:
maybe use a Record? Not sure if there would be much benefit but worth considering (I think there is only one other service that has experimented with records)

Reply:
potentially I could add this at a later date, I'll look into it :)